styles.css

```diff
@@ -0,0 +1,575 @@
+}
+
+.introduction {
+  margin: 0 15px;
+  background-image: url("images/Abrehot_main.jpeg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: right;
+}
+
+.introduction h1 {
+  color: #fff;
+}
+
+.introduction > h2 {
+  background-image: url(images/fire.jpg);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-size: 125px;
+  font-size: 52px !important;
+  font-weight: 900 !important;
+}
+
+.introduction div {
+  border: 2px solid white;
+  outline: 3px solid white;
+  font-family: Lato, sans-serif;
+}
+
+.tbm-header {
+  font-size: 32px;
+  font-weight: 600;
+  color: #5e5e5e;
+  text-align-last: left;
+  font-family: Lato, sans-serif;
+  padding-top: 7%;
+}
+
+.nyerere {
+  color: #767676;
+}
+
+form {
+  padding: 20px;
+  background-color: #ececec;
+}
+
+input:not([type="submit"]) {
+  border: none;
+  min-width: 300px;
+  width: 80%;
+}
+
+input[type="submit"] {
+  display: block;
+  margin: 20px auto 0;
+  font-family: Lato, sans-serif;
+  font-weight: bolder;
+  color: #ececec;
+  border: none;
+}
+
+.main-part {
+  background-image: url("images/x-background.png");
+  background-size: 20px;
+  background-color: #2d3037;
+}
+
+.main-part h2 {
+  color: #fff;
+  text-align: center;
+  padding-top: 30px;
+}
+
+.main-part img {
+  width: 80px;
+  height: 80px;
+  margin: 10px;
+}
+
+.main-part p {
+  margin: 10px;
+}
+
+.main-div {
+  margin: 20px;
+}
+
+.main-div div {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  background-color: rgb(60 63 70 / 50%);
+  margin: 10px 5px;
+}
+
+.main-div p {
+  text-align: justify;
+  color: #fff;
+}
+
+footer h3 {
+  font-family: Lato, sans-serif;
+  font-weight: 800;
+}
+
+.main-div h3 {
+  min-width: 70px;
+  color: #ff4b39;
+  margin: 10px;
+  font-weight: bolder;
+}
+
+.last-main-div {
+  padding: 50px;
+}
+
+.last-main-div h3 {
+  color: #fff;
+  padding: 30px;
+  margin: 0;
+}
+
+.partcipants img {
+  min-width: 150px;
+  max-width: 150px;
+  height: 150px;
+  margin: 10px;
+  object-fit: cover;
+  z-index: 1000;
+}
+
+.content {
+  margin: 10px;
+}
+
+.info {
+  text-align: justify;
+  color: red;
+}
+
+.investors button {
+  width: 80%;
+  margin: 10%;
+  padding: 20px;
+  border-color: #ececec;
+  background-color: white;
+  border: none;
+}
+
+.about-intro {
+  background-color: white;
+}
+
+.about-intro p {
+  text-align: justify;
+  text-align-last: center;
+  padding: 3%;
+}
+
+.about-main {
+  padding: 25px;
+  background-color: white;
+}
+
+.about-main h2 {
+  text-align-last: center;
+}
+
+.dash {
+  text-align: center;
+  color: red;
+  padding-bottom: 20px;
+}
+
+.about-main p {
+  text-align: justify;
+  padding: 0 6%;
+}
+
+.overlay {
+  position: relative;
+  margin: 20px;
+  background-color: rgb(245 122 89);
+}
+
+.overlay img {
+  width: 100%;
+  opacity: 0.4;
+}
+
+.overlay h3 {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
- [ ] I noticed you repeating the same code in two different places. So, how about we make a CSS class, add this code to it, and then use that code anywhere we want to apply the same style? This method avoids having to repeat the same code over and over. Not only will we not violate the `Do Not Repeat Yourself` rule `DRY` for short, but it will also save us from typing if we want to use the same style in a different location. :ok_hand: 😊

Like this :point_down: for example

```
positioning {
  position: absolute;
  left: 50%;
  top: 50%;
  transform: translate(-50%, -50%);
}
```
You can apply the same principle in all the places where you have repeating code. 🤓